doc/whatsnew/2/2.15/index.rst

```diff
@@ -42,3 +42,5 @@ Other Changes
 
 Internal changes
 ================
+
+* Added a check in CI for changelogs entry (#6688)
```

Comment:
The question here is should we keep the old format with:
```
* Fix a regression where messages with dash are not fully parsed

  Closes #3604
```
Or is ``* Fix a regression where messages with dash are not fully parsed (#3604)`` better ? Sometime we might want to be more detailed use multiple line ? Note that #3604 was an issue number but we would use the PR number now (It was just a commit at the time https://github.com/PyCQA/pylint/commit/ffb354aea057c25d9e48fa22da2840a450d99f3e but now we can't push to main so there's always a PR)

Reply:
Let's keep the old system. It brings some uniformity to all changelogs and is also easier to parse the data programatically.